src/projects/UoM.story.tsx

```diff
@@ -0,0 +1,672 @@
+import React from "react";
+import {
+  Box,
+  Flex,
+  Heading2,
+  Heading3,
+  Text,
+  Icon,
+  QuietButton,
+  PrimaryButton,
+  Table,
+  DescriptionList,
+  DescriptionGroup,
+  DescriptionTerm,
+  DescriptionDetails,
+  Tooltip,
+  Select,
+  Input,
+  Toggle,
+  IconicButton,
+  TruncatedText,
+} from "../index";
+import { Columns } from "../Table/Table.types";
+import styled from "styled-components";
+import { border, minWidth, width } from "styled-system";
+
+export default {
+  title: "Projects/UoM",
+};
+
+// Table columns for units of measure
+const unitsColumns: Columns<{}> = [
+  { label: "eaches", dataKey: "fromUnit" },
+  { label: "ea", dataKey: "toUnit" },
+  { label: "", dataKey: "factor" },
+  { label: "", dataKey: "rounding" },
+  { label: "", dataKey: "precision" },
+  { label: "", dataKey: "direction" },
+  { label: "", dataKey: "status" },
+  { label: "", dataKey: "lastUpdated" },
+  { label: "", dataKey: "actions" },
+  { label: "", dataKey: "conversion" },
+];
+
+const unitsRowData = [
+  {
+    fromUnit: "cases",
+    toUnit: "1",
+    factor: "cs",
+    rounding: "=",
+    precision: "1",
+    direction: "ea",
+    status: "=",
+    lastUpdated: "1",
+    actions: "ea",
+    conversion: "",
+    id: "r1",
+  },
+  {
+    fromUnit: "pallets",
+    toUnit: "1",
+    factor: "pl",
+    rounding: "=",
+    precision: "1",
+    direction: "cs",
+    status: "=",
+    lastUpdated: "1",
+    actions: "ea",
+    conversion: "",
+    id: "r2",
+  },
+];
+
+const RatioTable = styled(Table)`
+  border-collapse: collapse;
+  > tbody > tr {
+    border-bottom: 1px solid;
+    border-color: ${({ theme }) => theme.colors.lightGrey};
+  }
+
+  td, th{
+    padding: ${({ theme }) => theme.space.x2} ${({ theme }) => theme.space.x1};
+    vertical-align: top;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    
+  }
+
+  td:has(.customCell){
+    padding: ${({ theme }) => theme.space.x1};
+    overflow: visible;
+  }
+
+  td:nth-child(1), th:nth-child(1) {padding-left: ${({ theme }) => theme.space.x1};
+
+  tr.inactive td {
+    color: ${({ theme }) => theme.colors.grey};
+  }
+`;
+
+export const Default = () => (
+  <Box p="x3" style={{ border: "dotted 1px violet" }}>
+    <Flex justifyContent="space-between" alignItems="center" mb="x4">
+      <Heading2 mb="0">Units of measure</Heading2>
```

Comment:
You can use the `compact` prop to remove Heading margin
```suggestion
      <Heading2 compact>Units of measure</Heading2>
```